www/assets/de.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think it would be a better idea to write all translation keys in all caps, since they should be more considered to be constants, than normal strings. When you see them in a context, where they are not piped immediately (as in the AppComponent), you can't really tell, that this is not actually a string which is displayed, but rather a key for a value which is translated at runtime.
Actually I think this translation library is generally pretty poorly designed. Making these keys strings, can lead to a lot of errors which are very likely to not be found, right away, since they won't be found during compilation.
We could work around this, by using static classes, which maps static variables to the string keys. Like:
```ts
export class Translations {
  public static SERVER_ADRESS: string = 'server_address';
  public static USER: string = 'user';
  // ...
}
```
This way, you could use translation like `Translations.SERVER_ADRESS` which is checked at compile time, and therefore doesn't allow for typos.

However it also has the downside, that the translations class would have to be an attribute of every Component using translations, to be able to access it in the template, which adds additional overhead. But I still think this would be a cleaner solution. What do you think @dbabakow @mamahler @marcelhub ?